apps/web/src/services/asset-inventory/collector/collector-history/modules/NoCollectorModal.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
```

Comment:
question: Does this order of template writing also include the code conventions?

Reply:
No, we don't have any :)